modules/scheduler/scheduler/src/main/java/org/motechproject/scheduler/service/MotechSchedulerListener.java

```diff
@@ -0,0 +1,68 @@
+package org.motechproject.scheduler.service;
+
+import org.joda.time.DateTime;
+import org.motechproject.event.MotechEvent;
+import org.motechproject.event.listener.annotations.MotechListener;
+import org.motechproject.scheduler.contract.JobId;
+import org.motechproject.scheduler.contract.RepeatingJobId;
+import org.motechproject.scheduler.contract.RepeatingSchedulableJob;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Map;
+
+/**
+ * This service is used to schedule/unschedule jobs through the motech event.
+ */
+@Service
+public class MotechSchedulerListener {
+
+    private static final String SCHEDULE_JOB = "scheduleJob";
+    private static final String UNSCHEDULE_JOB = "unscheduleJob";
+
+    private static final String REPEAT_COUNT = "repeatCount";
+    private static final String REPEAT_INTERVAL_TIME = "repeatIntervalInSeconds";
+    private static final String JOB_SUBJECT = "jobSubject";
+
+    private MotechSchedulerService schedulerService;
+
+    /**
+     * Handles the motech event scheduling new job.
```

Comment:
Seems like it only schedules repeating job, can we update the javadoc to mention this?